src/Compilers/CSharp/Test/Syntax/IncrementalParsing/IncrementalParsingTests.cs

```diff
@@ -3019,6 +3019,50 @@ void M()
             WalkTreeAndVerify(tree.GetCompilationUnitRoot(), fullTree.GetCompilationUnitRoot());
         }
 
+        [Fact]
+        public void Lambda_EditAttributeList()
```

Comment:
Consider tests where the parameter list isn't parenthesized, where there wasn't an attribute to start with, and where the attribute is applied to a parameter, not the lambda.